libs/partners/upstage/langchain_upstage/layout_analysis.py

```diff
@@ -204,3 +204,45 @@ def lazy_load(self) -> Iterator[Document]:
                 exclude=self.exclude,
             )
             yield from parser.lazy_parse(blob)
+
+    def merge_and_split(
+        self, documents: List[Document], splitter: Optional[object] = None
```

Comment:
fyi if you want type hints you could add a dev dependency on text splitters and use the type